src/transport/tests/TestSessionManagerDispatch.cpp

```diff
@@ -165,6 +165,35 @@ struct MessageTestEntry theMessageTestVector[] = {
         .nonce        = "\x01\x78\x56\x34\x12\x01\x00\x00\x00\x00\x00\x00\x00",
         .privacyNonce = "\xdb\x7d\x23\x90\x0e\x9b\x3c\xe6\xd4\xbb\x03\x27\xd6",
 
+        .sessionId    = 0xdb7d, // 56189
+        .peerNodeId   = 0x0000000000000000ULL,
+        .groupId      = 2,
+        .sourceNodeId = 0x0000000000000002ULL,
+    },
+    {
```

Comment:
What about YAML tests? We should add a functional/end-to-end test to validate the feature here.

Reply:
This PR just adds decrypt capability for forward compatibility.  All nodes will still generate Pflag=0 messages however, so YAML tests are premature.